NEW_PROJECT_APPLICATION.md

```diff
@@ -8,6 +8,10 @@ Who is the main point of contact during the application process?
 
 A rough description of the project in less than 100 words.
 
+## Reason for Applying
+
+A short description of the reasons why your project is applying to join the foundation, and your expectations from the foundation.
```

Comment:
Specifying a word count would be useful.

Reply:
Any suggestions for a number? I'm really rather ambivalent on this. React to this with 🚀for 250, 👀for 100, or 👎to keep it unspecified.

Reply:
Having gone through the process myself, having a clear expectation of size was incredibly helpful.

Reply:
Based on feedback, I added a limit of 100 words.